README.md

```diff
@@ -1,4 +1,4 @@
-# Scalingo-CLI v1.24.2
+# Scalingo-CLI v1.25.0
```

Comment:
nitpick: I usually do a big search & replace in the README to replace all occurrences of 1.24.2 with 1.25.0 in the file. Here you didn't replaced the occurrences in the commands you are supposed to copy-paste for the release, i.e.

```
git switch --create release/1.24.2
git add .
git commit -m "Bump version 1.24.2"
git push --set-upstream origin release/1.24.2
gh pr create --reviewer=EtienneM --title "$(git log -1 --pretty=%B)"
```